doc/source/Getting_started/index.rst

```diff
@@ -79,6 +79,29 @@ faster resolution of complex simulations for design and optimization.
 For more information, see `Ansys Electronics <https://https://www.ansys.com/products/electronics>`_
 on the Ansys website.
 
+PyAEDT Cheat Sheets
+-------------------
+
+PyAEDT cheat sheet introduces the basics that you need to use PyAEDT.
+PyAEDT cheat sheet is one-page reference providing syntax rules and commands
+for using the PyAEDT API and EDB API:
+
+**PyAEDT Cheat Sheet:** `PyAEDT API <https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.pdf>`_
+
+**EDB Cheat Sheet:** `EDB API <https://cheatsheets.docs.pyansys.com/pyedb_API_cheat_sheet.pdf>`_
+
+
+Get Help
+--------
+
+**Development Issues:** For PyAEDT development-related matters (e.g., bug reports), please see
```

Comment:
```suggestion
**Development issues:** For PyAEDT development-related matters, see
```
I don't think we need to mention bug reports here as the sentence after the link says the page is used to report bugs and request new features.